src/main/java/se/sundsvall/customer/api/validation/impl/ValidCustomerDetailsRequestConstraintValidator.java

```diff
@@ -12,6 +12,6 @@ public class ValidCustomerDetailsRequestConstraintValidator implements Constrain
 
 	@Override
 	public boolean isValid(final CustomerDetailsRequest request, final ConstraintValidatorContext context) {
-		return !(isEmpty(request.getPartyId()) && isBlank(request.getCustomerEngagementOrgId()));
+		return !(isBlank(request.getCustomerEngagementOrgId()));
```

Comment:
Isn't this validator obsolete now that the `@ValidOrganizationNumber(nullable = true)` has been changed to not allow nulls (`@ValidOrganizationNumber`)?

Reply:
Yes.